src/components/structures/ToastContainer.tsx

```diff
@@ -73,16 +73,22 @@ export default class ToastContainer extends React.Component<{}, IState> {
                 key,
                 toastKey: key,
             });
-            toast = (<div className={toastClasses}>
-                <div className="mx_Toast_title">
-                    <h2>{ title }</h2>
-                    <span>{ countIndicator }</span>
-                </div>
-                <div className="mx_Toast_body">{ React.createElement(component, toastProps) }</div>
-            </div>);
+
+            const content = React.createElement(component, toastProps);
+
+            toast = supplyWholeBody
+                ? content
+                : <div className={toastClasses}>
+                    <div className="mx_Toast_title">
+                        <h2>{ title }</h2>
+                        <span>{ countIndicator }</span>
+                    </div>
+                    <div className="mx_Toast_body">{ content }</div>
+                </div>;
```

Comment:
The `supplyWholeBody` prop appear a bit odd to me.
Now that the `title` is optional wouldn't it work to conditionally display `mx_Toast_title` based on whether the prop exists or not and keep `mx_Toast_body` always

I'm probably the obvious here

Reply:
This is of course a possibility but the problem is that we need to style the whole thing, so we would need to pass in another `className` for the container and maybe the toast itself. So this just seemed simpler to do but it's not exactly nice, though I think the other thing wouldn't be either.

Perhaps there is a better way to reorganize all of this to make a bit simpler :thinking:

Reply:
Yeah, passing in another className seems to be clearer